ddp-workspace/projects/ddp-dsm-ui/src/app/participant-page/participant-page.component.ts

```diff
@@ -173,7 +173,7 @@ export class ParticipantPageComponent implements OnInit, OnDestroy, AfterViewChe
       window.scrollTo( 0, 0 );
       this.scrolled = true;
     }
-    this.validateEmailInput(this.participant.data.profile['email']);
+    this.validateEmailInput(this.participant.data.profile['email'] ? this.participant.data.profile['email'] : '');
```

Comment:
`this.participant.data.profile['email'] ? this.participant.data.profile['email'] : ''` 
can be shortened like
`this.participant.data.profile['email'] || ''`